linguistics/src/main/java/com/yahoo/language/significance/impl/DefaultSignificanceModelRegistry.java

```diff
@@ -23,24 +24,45 @@
 public class DefaultSignificanceModelRegistry implements SignificanceModelRegistry {
 
     private final Map<Language, SignificanceModel> models;
+
     @Inject
-    public DefaultSignificanceModelRegistry(SignificanceConfig cfg) { this(new Builder(cfg)); }
-    private DefaultSignificanceModelRegistry(Builder b) {
+    public DefaultSignificanceModelRegistry(SignificanceConfig cfg) {
         this.models = new EnumMap<>(Language.class);
-        b.models.forEach((language, path) -> {
-            models.put(language,
-                    uncheck(() -> new DefaultSignificanceModel(path)));
-        });
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Use the shared object mapper instance from `com.yahoo.json.Jackson.mapper()` instead.

Reply:
Nvm, the `container-core` is not a dependency of `linguistics`.